### DISCUSSION TITLE
[gatsby-source-wordpress] Cannot query field "wpGatsbyCompatibility" on type "Query"

### DISCUSSION THREAD

**Discussion by Acen** [gatsbyjs/gatsby#34844] [Help]
Hey there - having an issue with the check-plugin-requirements script attempting to [query apparently missing fields](https://reactjs.org/docs/error-decoder.html/?invariant=31&args%5B%5D=AggregateError%3A%20GraphQL%20Document%20Validation%20failed%20with%206%20errors%3B%0A%20%20Error%200%3A%20GraphQLDocumentError%3A%20Cannot%20query%20field%20%22wpGatsbyCompatibility%22%20on%20type%20%22Query%22.%0A%20%20%20%20at%20%2FProject%2Fnode_modules%2Fgatsby-source-wordpress%2Fdist%2Fsteps%2Fcheck-plugin-requirements.js%3A5%3A3%0A%0AError%201%3A%20GraphQLDocumentError%3A%20Cannot%20query%20field%20%22isWpGatsby%22%20on%20type%20%22Query%22.%0A%20%20%20%20at%20%2FProject%2Fnode_modules%2Fgatsby-source-wordpress%2Fdist%2Fsteps%2Fcheck-plugin-requirements.js%3A2%3A3%0A%0AError%202%3A%20GraphQLDocumentError%3A%20Cannot%20query%20field%20%22generalSettings%22%20on%20type%20%22Query%22.%0A%20%20%20%20at%20%2FProject%2Fnode_modules%2Fgatsby-source-wordpress%2Fdist%2Fsteps%2Fcheck-plugin-requirements.js%3A2%3A3%0A%0AError%203%3A%20GraphQLDocumentError%3A%20Cannot%20query%20field%20%22wpGatsby%22%20on%20type%20%22Query%22.%0A%20%20%20%20at%20%2FProject%2Fnode_modules%2Fgatsby-source-wordpress%2Fdist%2Fsteps%2Fcheck-plugin-requirements.js%3A5%3A3%0A%0AError%204%3A%20GraphQLDocumentError%3A%20Cannot%20query%20field%20%22schemaMd5%22%20on%20type%20%22Query%22.%0A%20%20%20%20at%20%2FProject%2Fnode_modules%2Fgatsby-source-wordpress%2Fdist%2Fsteps%2Fingest-remote-schema%2Fdiff-schemas.js%3A2%3A3%0A%0AError%205%3A%20GraphQLDocumentError%3A%20Cannot%20query%20field%20%22generalSettings%22%20on%20type%20%22Query%22.%0A%20%20%20%20at%20%2FProject%2Fnode_modules%2Fgatsby-source-wordpress%2Fdist%2Fsteps%2Fingest-remote-schema%2Fdiff-schemas.js%3A5%3A3).

```
gatsby info

  System:
    OS: macOS 12.0.1
    CPU: (8) arm64 Apple M1 Pro
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.2 - ~/.nvm/versions/node/v16.13.2/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v16.13.2/bin/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.2/bin/npm
  Languages:
    Python: 2.7.18 - /usr/bin/python
  Browsers:
    Chrome: 98.0.4758.102
    Safari: 15.1
  npmPackages:
    gatsby: ^4.7.2 => 4.7.2
    gatsby-plugin-graphql-codegen: ^3.1.1 => 3.1.1
    gatsby-plugin-image: ^2.7.0 => 2.7.0
    gatsby-plugin-manifest: ^4.7.0 => 4.7.0
    gatsby-plugin-offline: ^5.7.0 => 5.7.0
    gatsby-plugin-react-helmet: ^5.7.0 => 5.7.0
    gatsby-plugin-sharp: ^4.7.0 => 4.7.0
    gatsby-plugin-styled-components: ^5.7.0 => 5.7.0
    gatsby-plugin-ts: ^3.1.1 => 3.1.1
    gatsby-plugin-typescript: ^4.7.0 => 4.7.0
    gatsby-source-filesystem: ^4.7.0 => 4.7.0
    gatsby-source-wordpress: ^6.8.0-next.2 => 6.8.0-next.2
    gatsby-transformer-sharp: ^4.7.0 => 4.7.0
  npmGlobalPackages:
    gatsby-cli: 4.7.0
```

gatsby-source-wordpress config:
```   
 {
      resolve: `gatsby-source-wordpress`,
      options: {
        url:
          process.env.WPGRAPHQL_URL,
        schema: {
          //Prefixes all WP Types with "Wp" so "Post and allPost" become "WpPost and allWpPost".
          typePrefix: `Wp`,
        },
      },
    },
```

Any ideas?

**Comment by yq-ftadini** [comment]
i have the exact same problem
